open-sphere-plugins/infinity/src/main/java/io/opensphere/infinity/json/Histogram.java

```diff
@@ -0,0 +1,155 @@
+package io.opensphere.infinity.json;
+
+/** Elasticsearch histogram JSON bean. Used with numeric and date binning. */
+public class Histogram
+{
+    /** The field */
+    private String myField;
+
+    /** The bin interval */
+    private double myInterval;
+
+    /** The min_doc_count */
+    private int myMinDocCount;
+
+    /** The missing */
+    private long myMissing;
+
+    /** The bin offset */
+    private double myOffset;
+
+    /**
+     * Constructor
+     */
+    public Histogram()
+    {
```

Comment:
Document that this is intentionally blank (comment within the empty body)

Reply:
Updated.  I am not sure why this isn't showing here. https://github.com/ngageoint/opensphere-desktop/commit/3dafa702cd2fe9527121d18c0b971ddd2654ba12#diff-6